lib/Targets/AIETargets.cpp

```diff
@@ -314,7 +314,11 @@ SECTIONS
           int col = tileOp.colIndex();
           int row = tileOp.rowIndex();
           if (auto coreOp = tileOp.getCoreOp()) {
-            output << '(' << std::to_string(col) << ',' << std::to_string(row) << "),";
+            std::string elf_file = "None";
```

Comment:
I wonder if aiecc.py is going to rely on this mechanism to compute the names of the elf files, then maybe it should always use it?  basically this would push the 'corefile' function from aiecc.py into C++.